appcenter-data/Data.js

```diff
@@ -37,4 +39,16 @@ Data.ReadOptions = class {
     }
 };
 
+Data.WriteOptions = class {
+    constructor(timeToLive) {
+        this.timeToLive = timeToLive;
+    }
+};
+
+function convertTimestampToDate(result) {
+    // Create a new `Date` object from timestamp as milliseconds
+    result.lastUpdatedDate = new Date(result.lastUpdatedDate);
```

Comment:
Question: Can `result` be undefined or null? Do we handle falsy values gracefully here?

Reply:
It shouldn't be since I pass this function as a callback for a success. I added some error handling code just in to be safe.